ee/security-sinks/package.json

```diff
@@ -9,10 +9,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/boxyhq/jackson.git"
+    "url": "git+https://github.com/boxyhq/jackson.git"
   },
-  "main": "src/index.ts",
-  "types": "src/index.ts",
+  "main": "dist/index.js",
```

Comment:
This should be changed in ci/cd and not here. Jackson should use security-sinks directly.